src/server/schemas/nft/v5.ts

```diff
@@ -0,0 +1,79 @@
+import { Type } from "@sinclair/typebox";
+import { nftInputSchema } from ".";
+import { AddressSchema } from "../address";
+
+/**
+ * ERC721
+ */
+export const signature721InputSchemaV5 = Type.Object({
+  metadata: Type.Union([Type.String(), nftInputSchema]),
+  to: {
+    ...AddressSchema,
+    description:
+      "The wallet address that can use this signature to mint tokens. This is to prevent another wallet from intercepting the signature and using it to mint tokens for themselves.",
+  },
+  price: Type.Optional(
+    Type.String({
+      description:
+        'The amount of the "currency" token this token costs. Example: "0.1".',
+    }),
+  ),
+  priceInWei: Type.Optional(
+    Type.String({
+      description:
+        'The amount of the "currency" token this token costs in wei. Example: "100000000000000000".',
```

Comment:
this is a little ambiguous, USDC has 6 decimals, what does `wei` mean in this context?